run-bahmni.sh

```diff
@@ -45,40 +45,38 @@ function checkIfDirectoryIsCorrect {
 }
 
 function start {
-    if [ "$1" == ".env" ]; then
-        echo "Executing command: 'docker compose up -d' with 1.0.0 images"
-        echo "Starting Bahmni with default profile from .env file"
-    elif [ "$1" == ".env.dev" ]; then
+    if [ "$file" == ".env.dev" ]; then
```

Comment:
This should just be $file is not empty. If not empty, I will pass it as-is. Someone can create a file called: env.production or env.litev1 etc and pass that to run-bahmni.sh. So, the check should just be, if some param is passed, we will run the --env-file with that param. Else we will run without any --env-file.

Reply:
We have now set `.env` as the default and will take in any argument unless it's empty.
```
file=".env"
if ! [ "$1" == "" ]; then
    file="$1"
fi
```